cnf-tests/testimages/stress/Dockerfile

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Can we also create a `Dockerfile.openshift` for this one?

Reply:
Will do. Not sure I can do the same for the utils image as I don't know if all the dependencies required are there.

Reply:
Looks like it's possible. Tested in https://github.com/openshift-kni/cnf-features-deploy/pull/153
Added a Dockerfile.openshift for test utils too